src/api/java/baritone/api/Settings.java

```diff
@@ -81,6 +81,8 @@
      */
     public final Setting<Double> jumpPenalty = new Setting<>(2D);
 
+    public final Setting<Double> ylevelboxsize = new Setting<>(15D);
```

Comment:
come on the settings are CLEARLY camelCase capitalized, follow the pattern babe

every setting cLEARLY has a javadoc before it, add one